app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SponsorsFragment.java

```diff
@@ -34,17 +62,104 @@ public void onAttach(Context context) {
         getComponent().inject(this);
     }
 
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        Disposable disposable = viewModel.getSponsorships()
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(
+                        this::renderSponsorships,
+                        throwable -> Timber.tag(TAG).e(throwable, "Failed to show sponsors.")
+                );
+        compositeDisposable.add(disposable);
+    }
+
+
     @Nullable
     @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+            Bundle savedInstanceState) {
         binding = FragmentSponsorsBinding.inflate(inflater, container, false);
         binding.setViewModel(viewModel);
-
         initView();
         return binding.getRoot();
     }
 
+    @Override
+    public void onStop() {
+        super.onStop();
+        compositeDisposable.dispose();
+    }
+
     private void initView() {
-        //
+        adapter = new SponsorshipsAdapter(getContext());
+        binding.recyclerView.setAdapter(adapter);
+        binding.recyclerView.addItemDecoration(new DividerItemDecoration(getContext(), DividerItemDecoration.VERTICAL));
+        binding.recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
+    }
+
+    private void renderSponsorships(List<SponsorshipViewModel> sponsorships) {
+        adapter.addAllWithNotify(sponsorships);
+    }
+
+    private static class SponsorAdapter extends ArrayRecyclerAdapter<SponsorViewModel, BindingHolder<ViewSponsorCellBinding>>
+            implements SponsorViewModel.Callback {
```

Comment:
I think this `Callback` should be implemented to `SponsorsFragment`
because `onClickSponsor()` should be implemented in `SponsorsFragment`, not in `SponsorsAdapter`